cmd/prometheus/main.go

```diff
@@ -176,6 +176,9 @@ func (c *flagConfig) setFeatureListOptions(logger log.Logger) error {
 			case "extra-scrape-metrics":
 				c.scrape.ExtraMetrics = true
 				level.Info(logger).Log("msg", "Experimental additional scrape metrics")
+			case "scrape-cache-interner":
```

Comment:
Should this be given a more generic name? We might want to use the shared interner for the Prometheus Agent too.

Reply:
Sure. I went with `share-string-interner`.